frontend/src/components/TaskHeatmap.module.scss

```diff
@@ -42,7 +42,7 @@
   .xaxislabel {
     @extend .typography-pre-title;
     position: relative;
-    left: 68px;
+    left: 32px;
```

Comment:
This is very brittle at the moment, the label should be actually centred in the available space.